File: Java/Einsteiger/uebung3b.java
/*
2.  Bisher wurden die verschiedenen Werte stets direkt in das Programm eingegeben.
    In der Praxis ist es jedoch üblich, dass der Anwender diese eingibt.
    Dieser Vorgang ist in Java relativ kompliziert.
    Da diese Funktion jedoch auch bei den weiteren Beispielen nützlich ist, soll sie hier kurz vorgestellt werden.
    Die Einzelheiten werden dabei nicht erklärt – das folgt in den weiteren Kapiteln (vorwiegend in den Kapiteln VII und VIII). (Code vorgegeben)
    Sie können mit dem eigentlichen Programm beginnen.
    Dazu sollen Sie dem Nutzer eine kleine Rechenaufgabe stellen und ihn dazu auffordern, diese zu lösen.
    Überprüfen Sie nun mit einer if-Abfrage, ob der Anwender das richtige Ergebnis ermittelt hat und geben Sie eine entsprechende Meldung aus.

Bonacina, Michael. Java Programmieren: für Einsteiger: Der leichte Weg zum Java-Experten (2. Auflage: komplett neu verfasst) (Einfach Programmieren lernen 1) (German Edition) (S.73-74). Kindle-Version. 
*/

import java.io.*;

public class uebung3b
{
    public static void main(String[] args) throws Exception
    {
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);

        System.out.print("Wieviel ist 999 + 1? ");
        int x = Integer.parseInt(br.readLine());

        if (x == 1000)
        {
            System.out.println("Richtig!");
        }
        else
        {
            System.out.println("Falsch!");
        }
    }
}

File: C/Einsteiger/uebung3b.c
/*
2.  Schreiben Sie ein Programm, das den Anwender zur Eingabe von drei Wörtern auffordert.
    Nehmen Sie diese einzeln auf.
    Fügen Sie sie anschließend zu einer einzigen Zeichenkette zusammen.
    Achten Sie darauf, zwischen den einzelnen Wörtern ein Leerzeichen anzubringen.

Neumann, Markus. C Programmieren: für Einsteiger: Der leichte Weg zum C-Experten (Einfach Programmieren lernen 8) (German Edition) (S.72). Kindle-Version.
*/

#include <stdio.h>
#include <string.h>

void main ()
{
    char x[65];
    char a[21];
    char b[21];
    char c[21];

    printf("Geben Sie Wort 1 (max 20 Zeichen) ein: ");
    scanf("%s", &a);
    printf("Geben Sie Wort 2 (max 20 Zeichen) ein: ");
    scanf("%s", &b);
    printf("Geben Sie Wort 2 (max 20 Zeichen) ein: ");
    scanf("%s", &c);

    strcpy(x, a);
    strncat(x, " ", 2);
    strncat(x, b , 21);
    strncat(x, " ", 2);
    strncat(x, c , 21);

    printf("Text: %s \n", x);
}